### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by auslin-aot**
# Issue Tracking

JIRA: 
Issue Type: FEATURE
https://aottech.atlassian.net/browse/FWF-1791

# Changes
Added Logo styles to design system

# Screenshots
image

### CODE REVIEW

File: src/stories/Introduction.stories.mdx [hunk 1]
```diff
@@ -119,7 +119,40 @@ import StackAlt from './assets/stackalt.svg';
...
-[TODO]
+Logo is a symbol that identifies a business.
+
+<span class="formsflow-logo">
+<a href="https://149641023.v2.pressablecdn.com/wp-content/uploads/2022/05/Site_logo.png">
+<img src="https://149641023.v2.pressablecdn.com/wp-content/uploads/2022/05/Site_logo.png"/>
+</a>
+</span>
+
+<div className="subheading">Image Logo Usage</div>
```
Comment (sreehari-aot) [inline src/stories/Introduction.stories.mdx]: USAGE only

File: src/stories/Introduction.stories.mdx [hunk 2]
```diff
@@ -119,7 +119,40 @@ import StackAlt from './assets/stackalt.svg';
...
+
+<span class="formsflow-logo">
+<a href="https://app2.aot-technologies.com/logo.svg">
+<img src="https://app2.aot-technologies.com/logo.svg"/>
+</a>
+<div>formsflow.ai</div>
+</span>
+
+
+<div className="subheading">Image and Text Logo Usage</div>
```
Comment (sreehari-aot) [inline src/stories/Introduction.stories.mdx]: only USAGE is needed, if its image only the component should be displayed in such a manner
eg: if its image and logo display image and logo and specify the usage

File: src/css/style.css [hunk 3]
```diff
@@ -279,6 +279,23 @@
...
+  padding: 0.375rem; }
+
+.formsflow-logo img, .formsflow-logo a {
+  max-width: 100%;
+  height: 1.5rem; }
+
+.formsflow-logo div {
+  font-size: 1.2rem;
+  font-weight: 700;
+  color: black; }
```
Comment (sreehari-aot) [inline src/css/style.css]: use variables

File: src/css/style.css [hunk 4]
```diff
@@ -279,6 +279,23 @@
...
+  white-space: nowrap;
+  padding: 0.375rem; }
+
+.formsflow-logo img, .formsflow-logo a {
+  max-width: 100%;
+  height: 1.5rem; }
+
+.formsflow-logo div {
+  font-size: 1.2rem;
+  font-weight: 700;
```
Comment (sreehari-aot) [inline src/css/style.css]: define variables for font-weight for logo and use gere

File: src/css/components/_logo.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    
+    img, a{
+        max-width: 100%;  
+        height: 1.5rem;
+    }
+    
+    div{
+        font-size: 1.2rem;
+        font-weight: 700;
+        color: black;
```
Comment (sreehari-aot) [inline src/css/components/_logo.scss]: same use variable for colour

**Review by sreehari-aot — CHANGES_REQUESTED** [review]

File: src/stories/Introduction.stories.mdx (hunk 1 above)
Comment (auslin-aot) [reply src/stories/Introduction.stories.mdx]: done

File: src/stories/Introduction.stories.mdx (hunk 2 above)
Comment (auslin-aot) [reply src/stories/Introduction.stories.mdx]: done

File: src/css/components/_logo.scss (hunk 5 above)
Comment (auslin-aot) [reply src/css/components/_logo.scss]: Added variables.

**Review by sreehari-aot — APPROVED** [review]
Looks good to me

File: src/css/components/_logo.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
+/* Style the logo */
+.formsflow-logo{
+    opacity: 1;
```
Comment (shuhaib-aot) [inline src/css/components/_logo.scss]: opacity no  needed

File: src/css/components/_logo.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    
+    img, a{
+        max-width: 100%;  
+        height: 1.5rem;
+    }
+    
+    div{
+        font-size: $font-size-logo;
+        font-weight: $font-weight-logo;
+        color: $color-text-black;
```
Comment (shuhaib-aot) [inline src/css/components/_logo.scss]: need to bring a variable

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: src/css/components/_logo.scss (hunk 6 above)
Comment (auslin-aot) [reply src/css/components/_logo.scss]: Done

File: src/css/components/_logo.scss (hunk 7 above)
Comment (auslin-aot) [reply src/css/components/_logo.scss]: Done

**Review by sreehari-aot — APPROVED** [review]
Looks good to me

**Review by shuhaib-aot — APPROVED** [review]

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]
need to bring the text within anchor tag because we need to click in that also

**Review by shuhaib-aot — APPROVED** [review]

**Review by shuhaib-aot — APPROVED** [review]
looks good to me

**Pull request merged**